<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSkillModelToolClass</class>
 <widget class="QMainWindow" name="QtSkillModelToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1510</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skill Model Exporter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>831</width>
      <height>781</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gLayoutPreview"/>
   </widget>
   <widget class="QLineEdit" name="editPath">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>1120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>在此输入工程所在目录</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>18</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>48</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>皮肤ID</string>
    </property>
   </widget>
   <widget class="QListView" name="list_view_skins">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>68</y>
      <width>256</width>
      <height>760</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>48</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>技能ID</string>
    </property>
   </widget>
   <widget class="QListView" name="list_view_skills">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>68</y>
      <width>120</width>
      <height>760</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>英雄ID</string>
    </property>
   </widget>
   <widget class="QListView" name="list_view_heros">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>68</y>
      <width>256</width>
      <height>760</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtSkillModelTool.qrc"/>
 </resources>
 <connections/>
</ui>
